Automattic/vip-go-mu-plugins
# Jetpack to 10.5

## Description
* Changes the default version of Jetpack to 10.5
* Adds  backwards compatibility logic that prevents the user from using Jetpack 10.5 unless they are on Wordpress 5.8+

## Changelog Description
### Plugin Updated: Jetpack 10.5
The Default version of Jetpack is 10.5, in mu-plugins.

## Checklist


## Steps to Test
1. Check out PR.
1. Go to `wp-admin` > `Jetpack`
1. Verify Jetpack is 10.5
